app/serializers/framework_question_serializer.rb

```diff
@@ -11,6 +11,7 @@ class FrameworkQuestionSerializer
   attributes :key, :section, :question_type, :options, :response_type
 
   has_many_if_included :descendants, serializer: FrameworkQuestionSerializer, &:dependents
+  has_many_if_included :flags, serializer: FrameworkFlagSerializer, &:framework_flags
```

Comment:
When calling single assessment endpoints, and including the following: `?include=flags,responses.question.descendants.**` this is causing an n+1 query. It is also returning unnecessary flag data, for those endpoints making them much larger. Maybe there is something we can do with either different serializers or custom relationships?